app/boards/shields/microdox/microdox.keymap

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+/ {
+        keymap {
```

Comment:
We're trying to be sure that all stock keymaps include at a minimum the `&bt BT_CLR` binding, and ideally the profile selection `&bt BT_SEL 0` bindings in some layer to make that usable out of the box. Can we find a spot here in one of the empty spots for this?